BragiCode/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Intake extends SubsystemBase {
+  public TalonFX IntakeM1;
+  public TalonFX IntakeM2;
+  public TalonFX IntakeM3;
```

Comment:
The motors should have more descriptive names. We're going to want to know which motor is in which position.

Reply:
Renamed in several places.